### PR TITLE
[YOLO examples] torch quant CPU benchmark

### PR DESCRIPTION
**Pull request by bfineran**
supports loading a recipe and model from sparsezoo, applying that recipe to the model, and then possibly converting it to a quantized torch model to run on CPU.

`torch.quantization.convert` has no affect on normal FP32 models

### CODE REVIEW

File: examples/ultralytics-yolo/benchmark.py [hunk 1]
```diff
@@ -385,18 +399,38 @@ def _load_model(args) -> (Any, bool):
...
         else:
-            print("Using full precision")
             model.float()
+
+        if args.device == "cpu":
+            # convert any QAT models to fully quantized for CPU execution
+            try:
+                model = torch.quantization.convert(model)
+            except Exception:
+                pass
```
Comment (rahul-tuli) [inline examples/ultralytics-yolo/benchmark.py:430]: Thoughts on using the following idiom?

```suggestion
            with suppress(Exception):
                model = torch.quantization.convert(model)
```

`suppress` decorator can be imported from contextlib
`from contextlib import suppress`
Available from python3.4 onwards

**Review by rahul-tuli — CHANGES_REQUESTED** [review]
Looks good overall, left one small suggestion!

**Review by rahul-tuli — APPROVED** [review]
Good Work Ben! This PR has my

File: examples/ultralytics-yolo/benchmark.py [hunk 2]
```diff
@@ -271,6 +277,15 @@ def parse_args(arguments=None):
             "post-processing. Defaults to use standard YOLOv3/YOLOv5 anchors"
         ),
     )
+    parser.add_argument(
```
Comment (markurtz) [inline examples/ultralytics-yolo/benchmark.py:280]: the recipe, if any was used, is already present in the state dict exported from ultralytics. Check the loading logic that we have in the models/export.py class for yolov5. We shouldn't need to have the user supply a recipe here

**Review by markurtz — CHANGES_REQUESTED** [review]

**Comment by mgoin** [comment]
This also died 😢 a valiant death though

**Pull request closed without merging**